core/geometry/src/curve/internalContexts/MultiChainCollector.ts

```diff
@@ -44,7 +44,7 @@ export class MultiChainCollector {
   private _gapTolerance: number;
   /** end point snap tolerance (assumed to be as tight or tighter than gapTolerance) */
   private _snapTolerance: number;
-  /** tolerance for choosing Path or Loop. If undefined, always Path. */
+  /** planarity tolerance, used to determine whether to return a Path or Loop in `grabResult(true)`. If undefined, always Path. */
```

Comment:
Not a big deal but I prefer to start docs with upper case: `Planarity tolerance, ...`
Same for lines 41, 43, and 45

Reply:
Ah, missed that, thanks.